internal/service/verify/resource_voice_phrase_test.go

```diff
@@ -0,0 +1,200 @@
+		resource.TestCheckResourceAttr(resourceFullName, "name", name),
+		resource.TestMatchResourceAttr(resourceFullName, "created_at", validation.RFC3339Regexp),
+		resource.TestMatchResourceAttr(resourceFullName, "updated_at", validation.RFC3339Regexp),
+	)
+
+	updatedVoicePhrase := resource.ComposeTestCheckFunc(
+		resource.TestMatchResourceAttr(resourceFullName, "id", validation.P1ResourceIDRegexp),
+		resource.TestMatchResourceAttr(resourceFullName, "environment_id", validation.P1ResourceIDRegexp),
+		resource.TestCheckResourceAttr(resourceFullName, "name", updatedName),
+		resource.TestMatchResourceAttr(resourceFullName, "created_at", validation.RFC3339Regexp),
+		resource.TestMatchResourceAttr(resourceFullName, "updated_at", validation.RFC3339Regexp),
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheckEnvironment(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             testAccCheckVoicePhraseDestroy,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVoicePhrase_Initial(environmentName, licenseID, resourceName, name),
+				Check:  initialVoicePhrase,
+			},
+			{
+				Config:  testAccVoicePhrase_Initial(environmentName, licenseID, resourceName, name),
+				Destroy: true,
+			},
+			{
+				Config: testAccVoicePhrase_Update(environmentName, licenseID, resourceName, updatedName),
+				Check:  updatedVoicePhrase,
+			},
+			{
+				Config:  testAccVoicePhrase_Update(environmentName, licenseID, resourceName, updatedName),
+				Destroy: true,
+			},
+			// changes
+			{
+				Config: testAccVoicePhrase_Initial(environmentName, licenseID, resourceName, name),
+				Check:  initialVoicePhrase,
+			},
+			{
+				Config: testAccVoicePhrase_Update(environmentName, licenseID, resourceName, updatedName),
+				Check:  updatedVoicePhrase,
+			},
+			{
+				Config: testAccVoicePhrase_Initial(environmentName, licenseID, resourceName, name),
+				Check:  initialVoicePhrase,
+			},
+		},
+	})
+}
+
+func testAccVoicePhraseConfig_NewEnv(environmentName, licenseID, resourceName, name string) string {
+	return fmt.Sprintf(`
+		%[1]s
+
+resource "pingone_voice_phrase" "%[3]s" {
+  environment_id = pingone_environment.%[2]s.id
+  name           = "%[4]s"
+
+  depends_on = [pingone_environment.%[2]s]
+}`, acctest.MinimalSandboxEnvironment(environmentName, licenseID), environmentName, resourceName, name)
+}
+
+func testAccVoicePhrase_Initial(environmentName, licenseID, resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+
+resource "pingone_voice_phrase" "%[3]s" {
+  environment_id = pingone_environment.%[2]s.id
+  name           = "%[4]s"
+
+  depends_on = [pingone_environment.%[2]s]
+}`, acctest.MinimalSandboxEnvironment(environmentName, licenseID), environmentName, resourceName, name)
+}
+
+func testAccVoicePhrase_Update(environmentName, licenseID, resourceName, name string) string {
```

Comment:
This function probably isn't needed - is it a duplicate of `testAccVoicePhrase_Initial`?